pkg/apis/core/v1/storagecluster.go

```diff
@@ -533,7 +533,7 @@ type DataProviderSpec struct {
 type GitOpsSpec struct {
 	Name   string            `json:"name,omitempty"`
 	Type   string            `json:"type"`
```

Comment:
type in this case still can be empty string. do we add omitempty and validation as well?